diff --git a/node_modules/@microsoft/signalr/dist/esm/IHubProtocol.d.ts b/node_modules/@microsoft/signalr/dist/esm/IHubProtocol.d.ts
index 1249508..9c5c780 100644
--- a/node_modules/@microsoft/signalr/dist/esm/IHubProtocol.d.ts
+++ b/node_modules/@microsoft/signalr/dist/esm/IHubProtocol.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="node" />
 import { ILogger } from "./ILogger";
 import { TransferFormat } from "./ITransport";
 /** Defines the type of a Hub Message. */
@@ -133,7 +132,7 @@ export interface IHubProtocol {
      * @param {string | ArrayBuffer | Buffer} input A string, ArrayBuffer, or Buffer containing the serialized representation.
      * @param {ILogger} logger A logger that will be used to log messages that occur during parsing.
      */
-    parseMessages(input: string | ArrayBuffer | Buffer, logger: ILogger): HubMessage[];
+    parseMessages(input: string | ArrayBuffer | logger: ILogger): HubMessage[];
     /** Writes the specified {@link @microsoft/signalr.HubMessage} to a string or ArrayBuffer and returns it.
      *
      * If {@link @microsoft/signalr.IHubProtocol.transferFormat} is 'Text', the result of this method will be a string, otherwise it will be an ArrayBuffer.
